packages/preview/chem-par/0.0.1/src/chem-par.typ

```diff
@@ -0,0 +1,15 @@
+#import "stateful.typ": *
+#import "constants.typ": *
```

Comment:
Since your package uses `*` imports a lot, the user document might get polluted with many unused definitions. I would recommend either not having these exports/imports here (are they used?) or showing `#import "@preview/chem-par:0.0.1": chem-style` in the README instead of the star import.

Reply:
Entry point is lib.typ

Reply:
oops!